frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx

```diff
@@ -181,17 +181,27 @@ const subscriptionState = (state: SubscriptionState) => {
 
 const menuActions = [
   Kebab.factory.Edit,
-  (kind, obj) => ({
-    label: 'Remove Subscription',
-    callback: () => createUninstallOperatorModal({ k8sKill, k8sGet, k8sPatch, subscription: obj }),
-    accessReview: {
-      group: kind.apiGroup,
-      resource: kind.plural,
-      name: obj.metadata.name,
-      namespace: obj.metadata.namespace,
-      verb: 'delete',
-    },
-  }),
+  (kind, obj, { clusterServiceVersions }) => {
```

Comment:
I dont think the `ResourceKebob` handles a third arg:

```
// kebab.tsx
actions.map((a) => a(kindObj, resource)),
```
https://github.com/openshift/console/blob/a294f781273799c45a6a898605b5f8afcae70248/frontend/public/components/utils/kebab.tsx#L361

Reply:
@benjaminapetersen  Would that be the cause of this frontend test error?
```
TypeError: Cannot destructure property `clusterServiceVersions` of 'undefined' or 'null'.
      at Array.Object.<anonymous>.menuActions (packages/operator-
lifecycle-manager/src/components/subscription.tsx:108:1)
```

Reply:
Yup, this:
```
(kind, obj, { clusterServiceVersions }) => {
```
says "I get `kind`, `obj`, and a 3rd arg that I'll pluck out `clusterServiceVersion`.  but that 3rd object is never passed, so it is always `null/undefined`, and `null` never has `null.clusterServiceVersion`.

Reply:
Right, this won't work for the list view when fix #4916 merges.

Reply:
If you are wanting to pass the third property, then you need to update `kebab.tsx` to support it:

```
// you could update this, if you need to
actions.map((a) => a(kindObj, resource, csv)), 
// but it looks like _.get(obj, 'status.installedCSV'); installedCSV is already part of obj, so you may not need this.
```